third_party/move/move-compiler/src/parser/syntax.rs

```diff
@@ -2680,10 +2683,11 @@ fn parse_address_specifier(context: &mut Context) -> Result<AddressSpecifier, Bo
 //        | "struct" <StructDefName> "(" Comma<Type> ")" <Abilities>?";"
 //        | "enum" <StructDefName> <Abilities>? "{" Comma<EnumVariant> "}" (<Abilities> ";")?
 //      StructDefName =
-//          <Identifier> <OptionalTypeParameters>
+//          <Identifier> <StructTypeParameter>
```

Comment:
What is `StructTypeParameter`?

Reply:
Right above "parse_struct_type_parameters", although there should be a `?`